core/pom.xml

```diff
@@ -27,9 +27,9 @@
     </licenses>
 
     <properties>
-        <revision>1.0.0</revision>
-        <maven.compiler.source>19</maven.compiler.source>
-        <maven.compiler.target>19</maven.compiler.target>
+        <revision>1.1.0</revision>
```

Comment:
There's no need to bump up the version of the core since we don't really publish it anywhere

Reply:
Why does the core still have a pom?

Reply:
Discussed offline